Prog/Demographic trends.sas

```diff
@@ -0,0 +1,1036 @@
+		NETMIG2014
+		NETMIG2015
+		NETMIG2016
+		NETMIG2017
+		RESIDUAL2010
+		RESIDUAL2011
+		RESIDUAL2012
+		RESIDUAL2013
+		RESIDUAL2014
+		RESIDUAL2015
+		RESIDUAL2016
+		RESIDUAL2017
+		GQESTIMATESBASE2010
+		GQESTIMATES2010
+		GQESTIMATES2011
+		GQESTIMATES2012
+		GQESTIMATES2013
+		GQESTIMATES2014
+		GQESTIMATES2015
+		GQESTIMATES2016
+		GQESTIMATES2017
+		RBIRTH2011
+		RBIRTH2012
+		RBIRTH2013
+		RBIRTH2014
+		RBIRTH2015
+		RBIRTH2016
+		RBIRTH2017
+		RDEATH2011
+		RDEATH2012
+		RDEATH2013
+		RDEATH2014
+		RDEATH2015
+		RDEATH2016
+		RDEATH2017
+		RNATURALINC2011
+		RNATURALINC2012
+		RNATURALINC2013
+		RNATURALINC2014
+		RNATURALINC2015
+		RNATURALINC2016
+		RNATURALINC2017
+		RINTERNATIONALMIG2011
+		RINTERNATIONALMIG2012
+		RINTERNATIONALMIG2013
+		RINTERNATIONALMIG2014
+		RINTERNATIONALMIG2015
+		RINTERNATIONALMIG2016
+		RINTERNATIONALMIG2017
+		RDOMESTICMIG2011
+		RDOMESTICMIG2012
+		RDOMESTICMIG2013
+		RDOMESTICMIG2014
+		RDOMESTICMIG2015
+		RDOMESTICMIG2016
+		RDOMESTICMIG2017
+		RNETMIG2011
+		RNETMIG2012
+		RNETMIG2013
+		RNETMIG2014
+		RNETMIG2015
+		RNETMIG2016
+		RNETMIG2017
+
+	;
+
+	county2 = put(COUNTY,z3.);
+	state2 = put(STATE,z2.);
+	ucounty=  state2|| county2;
+
+	drop SUMLEV REGION DIVISION STNAME CTYNAME;
+
+	 if ucounty in ("11001","24017","24021","24031","24033","51013","51059","51107","51153","51510","51600","51610","51683","51685");
+run;
+
+data population (where= (ucounty in("11001","24017","24021","24031","24033","51013","51059","51107","51153","51510","51600","51610","51683","51685" )));
+set NCDB.Ncdb_master_update;
+keep ucounty Jurisdiction trctpop7 trctpop8 trctpop9 trctpop0 trctpop1 numhhs7 numhhs8 numhhs9 numhhs0 numhhs1;
+
+  if ucounty in ("11001") then Jurisdiction =1;
+  if ucounty  in ("24017") then Jurisdiction =2;
+  if ucounty  in ("24021") then Jurisdiction =3;
+  if ucounty  in ("24031") then Jurisdiction =4;
+  if ucounty  in ("24033") then Jurisdiction =5;
+  if ucounty  in ("51013") then Jurisdiction =6;
+  if ucounty  in ("51059", "51600", "51610") then Jurisdiction =7;
+  if ucounty  in ("51107") then Jurisdiction =8;
+  if ucounty  in ("51153", "51683", "51685") then Jurisdiction =9; 
+  if ucounty  in ("51510") then Jurisdiction =10; 
+
+run;
+
+proc sort data=population;
+by ucounty;
+run;
+
+data pop17;
+set Cen_population_estimates;
+keep ucounty Jurisdiction POPESTIMATE2017;
+  if ucounty in ("11001") then Jurisdiction =1;
+  if ucounty  in ("24017") then Jurisdiction =2;
+  if ucounty  in ("24021") then Jurisdiction =3;
+  if ucounty  in ("24031") then Jurisdiction =4;
+  if ucounty  in ("24033") then Jurisdiction =5;
+  if ucounty  in ("51013") then Jurisdiction =6;
+  if ucounty  in ("51059", "51600", "51610") then Jurisdiction =7;
+  if ucounty  in ("51107") then Jurisdiction =8;
+  if ucounty  in ("51153", "51683", "51685") then Jurisdiction =9; 
+  if ucounty  in ("51510") then Jurisdiction =10; 
+run;
+
+proc sort data=pop17;
+by ucounty;
+run;
+
+data populationtrend;
```

Comment:
In this section you are merging a tract-file with a county file one-to-many. This is causing the 2017 population to be wildly over calculated in the proc summary below.